### ISSUE TITLE
SystemStackError: stack level too deep

### ISSUE THREAD

**Issue by Bonias** [prograils/lit#50]
On fresh database/lit cache, after trying save model with validation errors like this

```
SomeModelWithValidations.create # => false
```

when I try to use I18n.t I get SystemStackError: stack level too deep:

```
I18n.t(:some_key) # => SystemStackError: stack level too deep
```

More details, and possible fix at https://github.com/Bonias/lit/commit/e4a71ed18fa747dd3dd0b6a0ad9a324cc604f8d6.

**Comment by mlitwiniuk** [comment]
Already in master - https://github.com/prograils/lit/commit/e4a71ed18fa747dd3dd0b6a0ad9a324cc604f8d6

**Comment by mlitwiniuk** [comment]
Fix was not merged in

**Comment by mlitwiniuk** [comment]
Fixed by #116
